src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  all: inherit;
+  box-sizing: border-box;
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 45px;
+  margin: 0 8px;
+}
+
+body {
+  margin: 0 8px;
+  /* background-color: #e5e5e5; */
+  /* font-family: Avenir, Helvetica, sans-serif; */
+}
+
+form {
+  margin: 0;
+}
+
+.input {
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+  margin-top: 20px;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
```

Comment:
Move `font-family: Avenir, sans-serif;` to the `body` selector.
And use `font-family: inherit` everywhere you need.